src/AltGoslingComponent.tsx

```diff
@@ -25,6 +26,17 @@ export const AltGoslingComponent = (props: AltGoslingCompProps) => {
         setTheme('dark')
     }
  
+    // Directly subscribe to the `rawData` event when the component is mounted
+    useEffect(() => {
+        // rawData
+        PubSub.subscribe("rawData", (_: string, data: {id: string, data: Datum[]}) => {
+            console.log("rawData", data);
+        });
```

Comment:
My understanding about the issue that we want to address is that, occasionally, `gosRef` instance is created **after** `rawData` events are published internally in `<GoslingComponent/>`. This makes it impossible to subscribe to the first `rawData` event because `gosRef` (The object used to subscribe to the event) is not created yet.

As a workaround, this PR uses `PubSub` directly in order to subscribe to the event without using `gosRef`.